api/api.proto

```diff
@@ -592,7 +609,25 @@ service OpenStorageAlerts {
     rpc Enumerate(SdkAlertsEnumerateRequest)
     returns (SdkAlertsEnumerateResponse) {
         option(google.api.http) = {
-                get: "/v1/alerts"
+                get: "/v1/alerts/enumerate"
```

Comment:
This should be `/v1/alerts`. When you do a GET on this endpoint it should return alerts.